### ISSUE TITLE
Component not mounting - Reason: TypeError: i.reactive is not a function

### ISSUE THREAD

**Issue by Kenvdb** [mya-ake/vue-window-size#421]
I'm using **Vue**, **@vue/composition-api** and **vue-window-size** but can't seem to get it working with the latest version of Vue Window Size. 

**package.json**
``` "dependencies": {
    "@gtm-support/vue2-gtm": "^1.2.1",
    "@sentry/browser": "^6.15.0",
    "@sentry/integrations": "^6.15.0",
    "@sentry/tracing": "^6.15.0",
    "@vue/composition-api": "^1.4.1",
    "axios": "^0.21.4",
    "axios-oauth-client": "^1.4.4",
    "axios-token-interceptor": "^0.2.0",
    "bootstrap": "^4.6.1",
    "bootstrap-vue": "^2.21.2",
    "clean-webpack-plugin": "^3.0.0",
    "core-js": "^3.19.3",
    "google-maps-api-loader": "^1.1.1",
    "gsap": "^3.8.0",
    "localforage": "^1.10.0",
    "lodash": "^4.17.21",
    "lottie-web": "^5.8.1",
    "vee-validate": "^3.4.14",
    "vue": "^2.6.14",
    "vue-i18n": "^8.26.7",
    "vue-router": "^3.5.3",
    "vue-window-size": "^1.1.2",
    "vue2-hammer": "^2.1.2",
    "vuex": "^3.6.2",
    "vuex-extensions": "^1.1.5"
  }
```

**Error**
```
__[Vue warn]: Failed to resolve async component: function DetectScreenSize() {
  return __webpack_require__.e(/*! import() | components */ "components").then(__webpack_require__.bind(null, /*! ./DetectScreenSize.vue */ "./resources/web/vue/components/DetectScreenSize.vue"));
}
Reason: TypeError: i.reactive is not a function__
```

It looks like it's missing reactive from the Composition API, but it's included in the main.js of the application:
```
import VueCompositionAPI from '@vue/composition-api'
Vue.use(VueCompositionAPI)
```

When I downgrade to 1.0.3 of of Vue Window Size, it's working as expected.

Any help would be nice.

**Comment by mya-ake** [comment]
@Kenvdb Thank you for the issue.
It is the expected behavior. Please see the [README](https://github.com/mya-ake/vue-window-size#for-vue-v2)

> You can install v1 using the following command.
> When using with vue-window-size v1 it depends on the @vue/composition-api. And requires Vue.js v2.6 or higher.
> note: v1.1.0 or later is for vue3 only

***

At the time Vue.js v3 was released, I decided it would be difficult to maintain compatibility with Vue.js v2.
So when I released v1.1.x, I stopped supporting Vue.js v2. Since this is a breaking change, it should have been vue-window-size v2, but due to my mistake, it was released as v1.1. I apologize for the confusion.

**Comment by mya-ake** [comment]
I close this PR. If the problem persists, please create a new Issue again.
